src/App.scss

```diff
@@ -1 +1,11 @@
-// styles go here
```

Comment:
Replace the line comment with a block comment for better readability. For example, use '/* styles go here */' instead of '// styles go here'.